### PR TITLE
Fabric 1.19

### PR DESCRIPTION
**Pull request by kosma**

### CODE REVIEW

File: fabric-1.19/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+minecraft_version=1.19-pre3
+yarn_mappings=1.19-pre3+build.4
+loader_version=0.14.6
+fabric_version=0.53.4+1.19
```
Comment (ThalusA) [inline fabric-1.19/gradle.properties]:
```suggestion
minecraft_version=1.19
yarn_mappings=1.19+build.1
loader_version=0.14.6
fabric_version=0.55.2+1.19
```

File: fabric-1.19/src/main/resources/fabric.mod.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ]
+  },
+  "mixins": [
+    "dynmap.mixins.json"
+  ],
+
+  "depends": {
+    "fabricloader": ">=0.14.6",
+    "fabric": ">=0.53.4",
+    "minecraft": "1.19.x"
```
Comment (ThalusA) [inline fabric-1.19/src/main/resources/fabric.mod.json]:
```suggestion
    "fabricloader": ">=0.14.6",
    "fabric": ">=0.55.2",
    "minecraft": "1.19.x"
```

File: fabric-1.19/src/main/resources/dynmap.mixins.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "required": true,
+  "minVersion": "0.8",
+  "package": "org.dynmap.fabric_1_19.mixin",
+  "compatibilityLevel": "JAVA_16",
```
Comment (ThalusA) [inline fabric-1.19/src/main/resources/dynmap.mixins.json]:
```suggestion
  "compatibilityLevel": "JAVA_17",
```
You now require JAVA_17 with 1.19 right ?

File: fabric-1.19/build.gradle [hunk 4]
```diff
@@ -0,0 +1,60 @@
+plugins {
+	// TODO: switch to a stable Loom release once 1.19 lands
+	id 'fabric-loom' version '0.12-SNAPSHOT'
```
Comment (ThalusA) [inline fabric-1.19/build.gradle:3]: 0.12-SNAPSHOT uses the latest version of 0.12 which means all futures patches (don't change the API at all) 
TODO might be not needed

Comment (kosma) [reply fabric-1.19/build.gradle:3]: I really prefer to pin dependency versions to avoid the build breaking :)

**Pull request merged**